modules/core/src/main/scala/org/tessellation/http/routes/TrustRoutes.scala

```diff
@@ -1,30 +1,52 @@
 package org.tessellation.http.routes
 
 import cats.effect.Async
+import cats.syntax.applicativeError._
 import cats.syntax.flatMap._
 
+import org.tessellation.domain.cluster.programs.TrustPush
 import org.tessellation.domain.trust.storage.TrustStorage
 import org.tessellation.ext.codecs.BinaryCodec._
 import org.tessellation.kryo.KryoSerializer
+import org.tessellation.schema.trust.InternalTrustUpdateBatch
+import org.tessellation.sdk.ext.http4s.refined.RefinedRequestDecoder
 
 import org.http4s._
-import org.http4s.circe.CirceEntityCodec.circeEntityEncoder
 import org.http4s.dsl.Http4sDsl
 import org.http4s.server.Router
 
 final case class TrustRoutes[F[_]: Async: KryoSerializer](
-  trustStorage: TrustStorage[F]
+  trustStorage: TrustStorage[F],
+  trustPush: TrustPush[F]
 ) extends Http4sDsl[F] {
   private[routes] val prefixPath = "/trust"
 
-  private val httpRoutes: HttpRoutes[F] = HttpRoutes.of[F] {
+  private val p2p: HttpRoutes[F] = HttpRoutes.of[F] {
     case GET -> Root =>
       trustStorage.getPublicTrust.flatMap { publicTrust =>
         Ok(publicTrust)
       }
   }
 
+  private val cli: HttpRoutes[F] = HttpRoutes.of[F] {
+    case req @ POST -> Root =>
```

Comment:
Now that trust is here I removed the additional "trust" path that we had when it was in cluster routes. The "trust" prefix is enough.

Reply:
cc: @ryle-goehausen /\